depthai_common_msgs/include/depthai_common_msgs/plugin/CameraBoardSocket.h

```diff
@@ -0,0 +1,6 @@
+#include <depthai-shared/datatype/CameraBoardSocket.hpp>
+
+//#define DEPTHAI_COMMON_MSGS_MESSAGE_CAMERABOARDSOCKET_PLUGIN_CLASS_BODY \
+//    MSGPACK_DEFINE()
+
+MSGPACK_ADD_ENUM(dai::CameraBoardSocket);
```

Comment:
Do we not need the header include for MSGPACK_*?

Reply:
Mmm, not necessary. I replaced enums necessary for publishers to raw data types.

Reply:
In that case, how does this macro (MSGPACK_ADD_ENUM) get expanded?